packages/modules/imodel-browser/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@itwin/imodel-browser-react",
   "description": "Components that let the user browse the iModels of a context and select one.",
   "repository": "https://github.com/iTwin/admin-components-react/tree/main/packages/modules/imodel-browser",
-  "version": "1.2.2",
+  "version": "1.2.3",
```

Comment:
Revert this change, the version will be bumped by the change management.

Reply:
Reverted the change